dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -272,6 +272,8 @@ class BGPPeerMgr(Manager):
             [
                 ("meta", "localhost/bgp_asn"),
                 ("neigmeta", ""),
+                ("local_addresses", ""),
+                ("interfaces", ""),
```

Comment:
clean up this?

Reply:
It's necessary that is same as the last comment.